src/frontend/js/modules/header/header.js

```diff
@@ -105,8 +105,11 @@
         replace: true,
         template: require('../../../views/modules/header/header.pug'),
         link: function(scope) {
-        // We need this second variable because matchmedia.on is called too many times,
-        // for instance when top or bottom bars of mobiles finish moving.
+          // Check if we need to show the advanced search box
+          scope.showAdvancedSearch = process.env.SHOW_ADVANCED_SEARCH === 'true';
```

Comment:
The search bar isn't merely an advanced search but also a basic search.

Reply:
Thanks. I updated it